bigbluebutton-html5/imports/ui/components/menu/styles.scss

```diff
@@ -5,8 +5,6 @@
 
 .option {
   line-height: 1;
-  margin-right: 1.65rem;
```

Comment:
In other spots we use conditionals in the scss file only. Would it be cleaner doing it that way? See https://github.com/bigbluebutton/bigbluebutton/pull/13092/files